presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraType.java

```diff
@@ -434,9 +435,12 @@ public boolean isSupportedPartitionKey()
             case DOUBLE:
             case INET:
             case INT:
+            case SMALLINT:
+            case TINYINT:
             case FLOAT:
             case DECIMAL:
             case TIMESTAMP:
+            case DATE:
```

Comment:
Please add tests where date, smallint and tinyint is used as partition key.

Reply:
I looked into https://github.com/prestosql/presto/pull/141 and did not find any tests added when the support for these types were added for the first time. I tested them locally by upgrading the Cassandra version in docker container as the 2.2 version does not support these types. Is it okay to upgrade the Cassandra version to 3.11 and add tests ?

Reply:
> Is it okay to upgrade the Cassandra version to 3.11 and add tests ?

yes, i think so. @ebyhr?

Reply:
Where are you going to add tests? The Cassandra version in the product test already supports the type as #141.

Reply:
Thanks for the clarification. I was referring to the test inside the Cassandra connector module. I just had a chance to look into the product tests and noticed many tests. I will add it to presto-product-tests and update this PR. Once again, Thanks a lot for your support since the beginning..

Reply:
@sundeeparun We upgraded Cassandra docker version to 2.2 in other commit. Could you rebase and add a test again?